app/src/main/res/drawable/border_personagens.xml

```diff
@@ -3,7 +3,7 @@
 
     <item>
         <shape android:shape="rectangle">
-            <solid android:color="#B30000 "/>
+            <solid android:color="#88B30000"/>
```

Comment:
É possível colocar no arquivo de colors? Se não for, tudo bem.

Reply:
É possível sim, eu já coloquei no colors. Contudo vale ressaltar que o desenvolvimento desse xml não é meu. Acabei alterando a cor sem requerer e esqueci de voltar. Esse xml é do Henrique, se for feito um review do todo o código temos mais códigos hardcode.
Mas eu desenvolvi um xml chamado circle_corner, que estava com a cor branca em hardcode e já ajustei. Obrigada :)